spec/models/featured_collection_list_spec.rb

```diff
@@ -3,16 +3,17 @@
 require 'rails_helper'
 
 RSpec.describe FeaturedCollectionList, type: :model do
+  subject { described_class.new }
   let(:user) { create(:user).tap { |u| u.add_role(:admin, Site.instance) } }
   let(:account) { create(:account) }
   let(:collection1) { create(:collection, user: user) }
   let(:collection2) { create(:collection, user: user) }
+  let!(:featured_collection_1) { create(:featured_collection, collection_id: collection1.id) }
```

Comment:
reviewing the lint errors, one of them is about the use of Let!. 

A few ideas... because it doesn't look like you are using these variables, you could move this into a before block instead. 

```ruby
  # let!(:featured_collection_1) { create(:featured_collection, collection_id: collection1.id) }
  # let!(:featured_collection_2) { create(:featured_collection, collection_id: collection2.id) }
  
  before do
    create(:featured_collection, collection_id: collection1.id)
    create(:featured_collection, collection_id: collection2.id)
  end
```

Otherwise you would have to call on the variable as the first step within your it block to activate it before your expectation.